### ISSUE TITLE
import paths update

### ISSUE THREAD

**Issue by venzen** [richardkiss/pycoinnet#15]
Imports in Pycoinnet do not match the location of modules in the current HEAD of the master branch of Pycoin.

Could I pull a different branch of pycoinnet or shall I go ahead and update imports via a PR?

**Comment by richardkiss** [comment]
Yeah, it's not in a great state at the moment. If you have PRs to improve it, I'll definitely take a look at merging. Maybe add a pinned dependency to the `setup.py` too.
